CHANGELOG.unreleased.md

```diff
@@ -284,6 +284,30 @@ foo;
 );
 ```
 
+### Javascript: Use function literals in arguments to detect function composition ([#6033] by [@brainkim])
+
+Previously, we used names of functions and methods to detect function
```

Comment:
We should mention the history here. That we used a hardcoded list of "fp" function names, and that users of non-fp code witht he same name were confused why their code broke on multiple lines. And that we’re now trying out a new heuristic that we’re excited about, and that people are welcome to provide feedback, etc. Kind of.

Reply:
I gave it a shot. I appreciate the idea that the CHANGELOG should be used to sell a specific feature and motivate users to upgrade. If you have further thoughts on the prose or language we should use lemme know.